### PR TITLE
Add the release notes for 0.99.5 and update the newsletter

### PR DESCRIPTION
**Pull request by trustin**
- Add the release notes for 0.99.5
- Update the newsletter
  - Add sign-up form to the newsletter
  - Update articles and release highlights
- Fix some style issues

### CODE REVIEW

File: site/src/pages/release-notes/0.99.5.mdx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  so that the client does not have to do a DNS lookup. #2674 #2679
+  - The IP address is retrieved from `defaultNonLoopbackIpV4Address()` of <type://SystemInfo>.
+
+## 🛠️ Bug fixes
+
+- Fixed a bug where more than one thread can consume from a streaming message. #2687 #2683
+- You will not get a `NullPointerException` anymore when starting a <type://JettyService> with DEBUG level
+  logging enabled. #2688 #2691
+- PING interval now works as expected even if idle timeout is disabled. #2702
+- A server-side connection is now closed immediately when PING fails. #2699
```
Comment (ikhoon) [inline site/src/pages/release-notes/0.99.5.mdx:105]: A client-side connection will be closed immediately too.
https://github.com/line/armeria/pull/2699/files#diff-e18570c3e534d61452c5286c7ec35704

Comment (trustin) [reply site/src/pages/release-notes/0.99.5.mdx]: Thanks! Fixed.

File: site/src/pages/news/20200514-newsletter-0.mdx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  - @nirvanarsc shows how he ran his Armeria-based Spring Boot application using Docker and Kubernetes.
+  - Learn about the basics of Reactive Streams, its industry adoption and interoperability.
+- [Lessons learned from the adoption of Armeria to LINE's authentication system](https://speakerdeck.com/line_developers/lesson-learned-from-the-adoption-of-armeria-to-lines-authentication-system?slide=8)
+  - @imasahiro tells about his successful migration from a home-grown HTTP/2 server based on Netty 4.1.0.Beta8 (!) to Armeria.
+
+## We need your comments
+
+Your opinion means a lot to us. Please let us know what you think about these proposals:
+
+- #2148 Provide a way to build an HTTP client request flently
```
Comment (syleeeee) [inline site/src/pages/news/20200514-newsletter-0.mdx:32]: did you mean 'fluently'?

Comment (trustin) [reply site/src/pages/news/20200514-newsletter-0.mdx:32]: Doh! Pushed the fix via 498d2dca530842bd070cb567ceffd07e4b256a98

**Pull request merged**